app/modules/organizations/models.py

```diff
@@ -138,23 +165,68 @@ class Organization(db.Model, HoustonModel, OrganizationEDMMixin):
         'OrganizationUserMembershipEnrollment', back_populates='organization'
     )
 
+    moderator_enrollments = db.relationship(
+        'OrganizationUserModeratorEnrollment', back_populates='organization'
+    )
+
+    owner_guid = db.Column(db.GUID, db.ForeignKey('user.guid'), index=True, nullable=True)
```

Comment:
After working against a constraint issue, I believe the best solution here is to mirror the existing data imported from EDM and resolve this issue in a future DB upgrade